pkg/bootstrapper/bootstrapper.go

```diff
@@ -482,12 +487,19 @@ func (wmcb *winNodeBootstrapper) generateInitialKubeletArgs(args map[string]stri
 	if cloudProvider, ok := args["cloud-provider"]; ok {
 		kubeletArgs = append(kubeletArgs, "--cloud-provider="+cloudProvider)
 	}
+	// set default verbosity for kubelet
+	kubeletVerbosity := KubeletDefaultVerbosity
+	// look for verbosity value in kubelet systemd unit file configuration
 	if v, ok := args["v"]; ok && v != "" {
-		kubeletArgs = append(kubeletArgs, "--v="+v)
-	} else {
-		// In case the verbosity argument is missing, use a default value
-		kubeletArgs = append(kubeletArgs, "--v="+"3")
+		// and update, if found
+		kubeletVerbosity = v
+	}
+	// set the verbosity value from the program argument if any, this takes precedence
+	if wmcb.kubeletVerbosity != "" {
+		kubeletVerbosity = wmcb.kubeletVerbosity
```

Comment:
This looks confusing. You should key off `wmcb.kubeletVerbosity` being set or not. 
```suggestion
	// set default verbosity for kubelet
	kubeletVerbosity := KubeletDefaultVerbosity
    // set the verbosity value from the program argument if any, this takes precedence
	if wmcb.kubeletVerbosity != "" {
		kubeletVerbosity = wmcb.kubeletVerbosity
	} else {
    	// look for verbosity value in kubelet systemd unit file configuration
    	if v, ok := args["v"]; ok && v != "" {
	    	// and update, if found
		    kubeletVerbosity = v
    	}
    }
```
You shouldn't have to look at the systemd unit if the verbosity is specified.

Reply:
Good catch! Addressed in 1637182a088084487a20c257f9846789231ee89d